docs/creating_new_react_component.md

```diff
@@ -0,0 +1,95 @@
+# Creating a new React Component for v3's website
+
+## Create the component
+
+- Each component should be in its own directory
+- Should be placed in `/app/javascript/components/<componentname>`
+
+## Create the tests
+
+- Place tests in `/test/javascript/components/<component-name>/ComponentName.test.tsx`
```

Comment:
I suggest prefixing/suffixing `ComponentName` with `$`, e.g. `$ComponentName$`, to make it clear that it's a placeholder. And doing the same for `$component_name$` further down. A different placeholder syntax would be fine too as long as it conveys the message 🙂

Reply:
I am usually a fan of `<` and `>` as placeholders, but feel free to use whatever placeholder you prefer.

Reply:
Yeah, `<>` are better than what I suggested!